src/components/layout/Shell.tsx

```diff
@@ -42,39 +45,38 @@ export function Shell(props: Props) {
           >
             <Image src="/metabase-logo-with-wordmark.svg" />
 
-            <Box className="md:hidden">
+            <Flex display={{ base: "flex", sm: "none" }}>
               <Burger
                 opened={isMobileNavOpen}
                 onClick={toggleMobileNav}
                 aria-label="Toggle navigation"
                 color="#eee"
               />
-            </Box>
+            </Flex>
 
-            <Box className="hidden md:block">
+            <Flex display={{ base: "none", sm: "flex" }}>
               <SiteSwitcher />
-            </Box>
+            </Flex>
           </Flex>
 
           {isMobileNavOpen && (
-            <Flex className="md:hidden" bg="#2B2F32" px="16px" pb="8px">
+            <Flex
+              display={{ base: "flex", sm: "none" }}
+              bg="#2B2F32"
+              px="16px"
+              py="8px"
+            >
               <SiteSwitcher />
             </Flex>
           )}
         </AppShell.Header>
 
-        <AppShell.Navbar
-          p="md"
-          withBorder={false}
-          pt={107}
-          pl="30px"
-          zIndex={3}
```

Comment:
I'm not sure if this is addressing the z-index issue, but I still see it on a tablet view.
<img width="619" alt="image" src="https://github.com/user-attachments/assets/17be000d-2953-468d-9a21-47035f93e8d7">

Reply:
The links to "new dashboard" is not supposed to show up on tablet view, strange. z-index is quite tricky...

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/faf78d2757f972e0914d324a17d1758e9ec119de - the zIndex was removed because I thought it was only needed for the "See more" button in theStitch (which I fixed in this PR), but turns out I still need it for the New Dashboard modal to be on top of the navbar.

![CleanShot 2567-10-08 at 03 02 14@2x](https://github.com/user-attachments/assets/0e982afd-3cf9-411d-865c-8c5c0e791f70)